src/main/kotlin/models/Message.kt

```diff
@@ -1,14 +1,11 @@
 package models
 
-import org.jetbrains.exposed.sql.Table
-
-object Message : Table() {
-    val id = varchar("id", 36).primaryKey()
+object Message : BaseTable() {
     val channelId = varchar("channelId", 50)
-    val channelDisplayName = varchar("channelDisplayName", 50)
-    val channelType = varchar("channelType", 50)
+    //val channelDisplayName = varchar("channelDisplayName", 50)
```

Comment:
Should we just get rid of these?

Reply:
idk i thought u did those so, i just left them lol

Reply:
i didnt remember the context lol